chroma-common/Makefile

```diff
@@ -0,0 +1,40 @@
+include ../include/Makefile.version
```

Comment:
I also have a branch where I have started this kind of (separation of the Chroma python modules) work.  We should make sure not to duplicate each others' work.